saleor/dashboard/staff/forms.py

```diff
@@ -15,3 +16,12 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = User
         fields = ['email', 'groups', 'is_staff', 'is_active']
+        labels = {
+            'email': pgettext_lazy(
+                'Staff form label', 'Email'),
+            'groups': pgettext_lazy(
+                'Staff form label', 'Groups'),
+            'is_active': pgettext_lazy(
+                'Staff form label', 'Active'),
```

Comment:
Such boolean fields could have more readable and understandable labels. In this case, we could have "User is active".